### ISSUE TITLE
Issue with local run

### ISSUE THREAD

**Issue by brat002** [crumbhole/argocd-lovely-plugin#203]
We use relative paths to dependencies of helm charts. Looks like the plugin doesn't work correctly with them:
```
$ cat Chart.yaml
apiVersion: v2
name: pomerium
description: A Helm chart for Kubernetes
type: application
version: 33.0.1
appVersion: "24.0.0"

dependencies:
  - name: pomerium
    version: "27.0.2"
    repository: "file://../../generic-charts/chart_deps/pomerium/"
  - name: genapp
    alias: pomerium-common
    version: ">0.1.0, <0.2.0"
    repository: "oci://asia-docker.pkg.dev/xxxx-xxxx/gcp-helm-repository/stable"

$ helm dep update . --skip-refresh
Saving 2 charts
Downloading genapp from repo oci://asia-docker.pkg.dev/xxxx-xxxx/gcp-helm-repository/stable
Pulled: asia-docker.pkg.dev/xxxx-xxxx/gcp-helm-repository/stable/xxxx:0.1.262
Digest: sha256:006b8e3c2b2cbc392faea28b4d92e6f3386a148d7f1a3b418034783afd4ec141
Deleting outdated charts

$ argocd-lovely-plugin 
2023/06/17 10:23:39 exit status 1: Error: directory ../../generic-charts/chart_deps/pomerium not found

$ ls ../../generic-charts/chart_deps/pomerium
Chart.lock	Chart.yaml	OWNERS		README.md	charts		ci		scripts		templates	values.yaml
```

Version of plugin: argocd-lovely-plugin-0.19.4-darwin-amd64.
Please take a look.

**Comment by Joibel** [comment]
If you set LOVELY_ALLOW_GITCHECKOUT=true then it will do what you hope.

Lovely can modify the input files via MERGE, PATCH and PREPROCESSORS, so by default it takes a copy of the directory it is working in, but not the whole repo.

I'll leave this open and do an explainer.

**Comment by Joibel** [comment]
Closed by #209
